src/req.js

```diff
@@ -16,7 +16,7 @@ async function req(name, rootFile = __filename) {
   let url = createRequire(rootFile).resolve(name)
 
   try {
-    return (await import(pathToFileURL(url).href)).default
+    return (await import(`${pathToFileURL(url).href}?t=${Date.now()}`)).default
```

Comment:
.href can be removed, template literal will call toString by default which returns href

Reply:
Done